bread/views/browse.py

```diff
@@ -67,23 +67,28 @@ class BrowseView(BreadView, LoginRequiredMixin, PermissionListMixin, ListView):
     orderingurlparameter: str = "ordering"
     objectids_urlparameter: str = "_selected"  # see bread/static/js/main.js:submitbulkaction and bread/layout/components/datatable.py
     bulkaction_urlparameter: str = "_bulkaction"
-    items_per_page_options: Tuple[int] = None
+    items_per_page_options: Optional[Tuple[int]] = None
     itemsperpage_urlparameter: str = "itemsperpage"
 
     title: Optional[hg.BaseElement] = None
-    columns: Tuple[Union[str, layout.datatable.DataTableColumn]] = ["__all__"]
+    columns: Tuple[Union[str, layout.datatable.DataTableColumn]] = ("__all__",)
```

Comment:
This should either be ```Tuple[Union[str, layout.datatable.DataTableColumn], ...]``` or maybe better ```Iterable[Union[str, layout.datatable.DataTableColumn]]```. We could restrict it to tuples, but allowing array syntax would make sense.

Okay, to be fully compliant we would need to rewrite ```bread.layout.datatable.DataTable.from_model``` to iterate only once over ```columns``` to allow any iterable (e.g. a generator). Which is possible without problems I think. Or we use ```Sequence[Union[str, layout.datatable.DataTableColumn]]``` instead, but that would feel like an arbitrary limitation.